README.md

```diff
@@ -28,7 +28,7 @@ $ git clone https://github.com/ovh/manager.git
 # Go to the project root
 $ cd manager
 
-# If you using nvm
+# If you are using nvm
```

Comment:
Good catch!

Same content located here:
https://github.com/ovh/manager/blob/master/docs/docs/guide/getting-started.md?plain=1#L21

Maybe we can apply this patch also.